terps/tads/glk/osfrobtads.h

```diff
@@ -34,7 +34,13 @@ extern "C++" {
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <unistd.h>
+#ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
+#define NOMINMAX
+#include <Windows.h>
```

Comment:
This is another case of not building on MinGW, although it seems to be a case issue there. MinGW calls this `windows.h`, not `Windows.h`.  Since Windows is case insensitive, I guess it's safe to rename this as lowercase.